src/tests/get_test_evidence_data.py

```
@@ -0,0 +1,132 @@
+import json
+import os
+import random
+from typing import Dict, List
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)